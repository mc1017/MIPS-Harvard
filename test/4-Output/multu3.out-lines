CPU : OUT :  54788096
