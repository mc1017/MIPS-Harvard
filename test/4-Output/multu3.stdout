CPU : OUT :  54788096
TB : Finished : active=0
test/1-ArithmeticandLS/multu3.v:59: $finish called at 100 (1s)
